File: app/src/main/java/com/startogamu/zikobot/core/module/spotify_api/model/Image.java
package com.startogamu.zikobot.core.module.spotify_api.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.parceler.Parcel;

import lombok.AllArgsConstructor;
import lombok.Data;

@Parcel
@Data
@AllArgsConstructor(suppressConstructorProperties = true)
public class Image {

    @SerializedName("height")
    @Expose
    public Integer height;
    @SerializedName("url")
    @Expose
    public String url;
    @SerializedName("width")
    @Expose
    public Integer width;

    public Image() {
    }
}

File: data/src/main/java/com/joxad/zikobot/data/module/spotify_api/model/Image.kt
package com.joxad.zikobot.data.module.spotify_api.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import lombok.AllArgsConstructor
import lombok.Data

class Image {

    @SerializedName("height")
    @Expose
    var height: Int? = null
    @SerializedName("url")
    @Expose
    var url: String? = null
    @SerializedName("width")
    @Expose
    var width: Int? = null
}
